### PR TITLE
slow_query: update the doc

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
- Add the following fields: `Optimize_time`, `Wait_TS`, `Index_names`, `Stats`, `Plan_from_binding`, `Has_more_results`, `KV_total`, `PD_total`, `Backoff_total`, `Write_sql_response_total`, `Result_rows`, `IsExplicitTxn` `Host` `Num_cop_tasks`
- Fix a typo `transaction_ts`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8801
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: identify-slow-queries.md [hunk 1]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
 * `Is_internal`: Whether a SQL statement is TiDB internal. `true` indicates that a SQL statement is executed internally in TiDB and `false` indicates that a SQL statement is executed by the user.
+* `Index_names`: The index names used by the statement.
+* `Stats`: The health state of tables. `pseudo` indicates that the statistics is unhealthy.
 * `Index_ids`: The IDs of the indexes involved in a statement.
 * `Succ`: Whether a statement is executed successfully.
 * `Backoff_time`: The waiting time before retry when a statement encounters errors that require a retry. The common errors as such include: `lock occurs`, `Region split`, and `tikv server is busy`.
 * `Plan`: The execution plan of the statement. Use the `select tidb_decode_plan('xxx...')` statement to parse the specific execution plan.
 * `Prepared`: Whether this statement is a `Prepare` or `Execute` request or not.
 * `Plan_from_cache`: Whether this statement hits the execution plan cache.
+* `Plan_from_binding`: Whether this statement has bound execution plans.
```
Comment (crazycs520) [inline identify-slow-queries.md:74]:
```suggestion
* `Plan_from_binding`: Whether this statement uses the bound execution plans.
```

File: identify-slow-queries.md [hunk 2]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
 * `Parse_time`: The parsing time for the statement.
 * `Compile_time`: The duration of the query optimization.
+* `Optimize_time`: The optimization time for the query plan.
+* `Wait_TS`: The waiting time of the transaction to get timestamps.
 * `Query`: A SQL statement. `Query` is not printed in the slow log, but the corresponding field is called `Query` after the slow log is mapped to the memory table.
 * `Digest`: The fingerprint of the SQL statement.
 * `Txn_start_ts`: The start timestamp and the unique ID of a transaction. You can use this value to search for the transaction-related logs.
 * `Is_internal`: Whether a SQL statement is TiDB internal. `true` indicates that a SQL statement is executed internally in TiDB and `false` indicates that a SQL statement is executed by the user.
+* `Index_names`: The index names used by the statement.
+* `Stats`: The health state of tables. `pseudo` indicates that the statistics is unhealthy.
```
Comment (TomShawn) [inline identify-slow-queries.md:67]:
```suggestion
* `Stats`: The health state of the involved tables. `pseudo` indicates that the statistics are unhealthy.
```

File: identify-slow-queries.md [hunk 3]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
 * `Plan_from_cache`: Whether this statement hits the execution plan cache.
+* `Plan_from_binding`: Whether this statement uses the bound execution plans.
+* `Has_more_results`: Whether this statement has more results to be fetched by users.
 * `Rewrite_time`: The time consumed for rewriting the query of this statement.
 * `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
 * `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
 * `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
 * `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.
+* `KV_total`: The time consumed for all the RPC requests on TiKV/TiFlash.
```
Comment (TomShawn) [inline identify-slow-queries.md:81]:
```suggestion
* `KV_total`: The time consumed on all the RPC requests on TiKV/TiFlash.
```

File: identify-slow-queries.md [hunk 4]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
+* `PD_total`: The time consumed for all the RPC requests on PD.
```
Comment (TomShawn) [inline identify-slow-queries.md:82]: consumed on

File: identify-slow-queries.md [hunk 5]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
+* `Backoff_total`: The time consumed for all the backoff in the execution of this statement.
+* `Write_sql_response_total`: The time consumed for sending results to the client.
+* `Result_rows`: The row count of the result.
+* `IsExplicitTxn`: Whether this statement is in an explicit transaction. If the value is `false`, the transaction is `autocommit=1` and the statement is committed after execution.
```
Comment (TomShawn) [inline identify-slow-queries.md:86]:
```suggestion
* `IsExplicitTxn`: Whether this statement is in an explicit transaction. If the value is `false`, the transaction is `autocommit=1` and the statement is automatically committed after execution.
```

File: identify-slow-queries.md [hunk 6]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
 * `Rewrite_time`: The time consumed for rewriting the query of this statement.
 * `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
 * `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
 * `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
 * `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.
+* `KV_total`: The time spent on all the RPC requests on TiKV/TiFlash.
+* `PD_total`: The time consumed for all the RPC requests on PD.
+* `Backoff_total`: The time consumed for all the backoff in the execution of this statement.
+* `Write_sql_response_total`: The time consumed for sending results to the client.
```
Comment (TomShawn) [inline identify-slow-queries.md:84]: time spent on

File: identify-slow-queries.md [hunk 7]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
+* `Plan_from_binding`: Whether this statement uses the bound execution plans.
+* `Has_more_results`: Whether this statement has more results to be fetched by users.
 * `Rewrite_time`: The time consumed for rewriting the query of this statement.
 * `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
 * `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
 * `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
 * `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.
+* `KV_total`: The time spent on all the RPC requests on TiKV/TiFlash.
+* `PD_total`: The time consumed for all the RPC requests on PD.
```
Comment (Oreoxmt) [inline identify-slow-queries.md:82]:
```suggestion
* `PD_total`: The time spent on all the RPC requests on PD.
```

File: identify-slow-queries.md [hunk 8]
```diff
@@ -55,21 +57,33 @@ Slow query basics:
...
+* `Backoff_total`: The time consumed for all the backoff in the execution of this statement.
```
Comment (Oreoxmt) [inline identify-slow-queries.md:83]:
```suggestion
* `Backoff_total`: The time spent on all the backoff in the execution of this statement.
```

File: identify-slow-queries.md (hunk 6 above)
Comment (Oreoxmt) [inline identify-slow-queries.md:84]:
```suggestion
* `Write_sql_response_total`: The time spent on sending results to the client.
```

File: identify-slow-queries.md [hunk 9]
```diff
@@ -55,22 +57,33 @@ Slow query basics:
 * `Query_time`: The execution time of a statement.
 * `Parse_time`: The parsing time for the statement.
 * `Compile_time`: The duration of the query optimization.
+* `Optimize_time`: The time consumed for optimizing the execution plan.
+* `Wait_TS`: The waiting time of the transaction to get timestamps.
```
Comment (TomShawn) [inline identify-slow-queries.md:61]:
```suggestion
* `Wait_TS`: The waiting time of the statement to get transaction timestamps.
```

File: identify-slow-queries.md [hunk 10]
```diff
@@ -55,22 +57,33 @@ Slow query basics:
...
 * `Rewrite_time`: The time consumed for rewriting the query of this statement.
 * `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
 * `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
 * `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
 * `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.
+* `KV_total`: The time spent on all the RPC requests on TiKV or TiFlash by this statement.
+* `PD_total`: The time spent on all the RPC requests on PD by this statement.
+* `Backoff_total`: time spent on all the backoff during the execution of this statement.
```
Comment (TomShawn) [inline identify-slow-queries.md:83]:
```suggestion
* `Backoff_total`: The time spent on all the backoff during the execution of this statement.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**